dir.traversal.targets

```diff
@@ -197,6 +197,10 @@
 
   <Target Name="GetFilesToPackage" DependsOnTargets="FilterProjects"
           Returns="@(FilesToPackage)">
+    <PropertyGroup>
+      <BuildInParallel Condition="'$(BuildInParallel)' == '' AND '$(MSBuildNodeCount)' > '1'">true</BuildInParallel>
```

Comment:
I had it set to true to begin with, but switched to this based on some code I used in buildtools before: https://github.com/dotnet/buildtools/pull/625#r59086676

Could have sworn I tested with setting true here and found that it overrode when someone explicitly asked to not build in parallel, but I trust you are correct.  Updated the PR.